OptimizelySDK/Utils/Validator.cs

```diff
@@ -43,6 +45,13 @@ public static bool ValidateJSONSchema(string configJson, string schemaJson = nul
                 return false;
             }
         }
+#endif
+#if NET35
+        public static bool ValidateJSONSchema(string configJson, string schemaJson = null)
+        {
+            return true;
```

Comment:
Yes, actually the Newsoft.Json.Schema doesn't have a support of 3.5 framework. That's why we skipped schema validation for 3.5, discussed earlier with Ali & Randall. @rjeike 
http://www.newtonsoft.com/jsonschema/help/html/Introduction.htm